src/Tizen.Applications.AttachPanel/Tizen.Applications.AttachPanel/AttachPanel.cs

```diff
@@ -13,29 +13,40 @@ public partial class AttachPanel
         /// </summary>
         /// <since_tizen> 4 </since_tizen>
         /// <param name="conformant">The caller's conformant.</param>
+        /// <exception cref="ArgumentNullException">Thrown when the parameter is null</exception>
         /// <exception cref="OutOfMemoryException">Thrown when an attempt to allocate the memory fails.</exception>
         /// <exception cref="InvalidOperationException">Thrown when the AttachPanel already exists or the <paramref name="conformant"/> is not a conformant object.</exception>
+        /// <exception cref="NotSupportedException">Thrown when the AttachPanel is not supported in the device.</exception>
         public AttachPanel(EvasObject conformant)
         {
             if (conformant == IntPtr.Zero)
             {
-                throw new ArgumentNullException("Use the value property, not null value");
+                throw new ArgumentNullException("Invalid conformant, it's null");
             }
 
-            IntPtr candidateAttachPanel = new IntPtr();
-            Interop.AttachPanel.ErrorCode err = Interop.AttachPanel.CreateAttachPanel(conformant, ref candidateAttachPanel);
+            if (IsAttachPanelSupported() == false)
+            {
+                CheckException(Interop.AttachPanel.ErrorCode.NotSupported);
+            }
+
+            if (IsInitialized())
+            {
+                CheckException(Interop.AttachPanel.ErrorCode.AlreadyExists);
+            }
+
+            var candidateAttachPanel = new IntPtr();
```

Comment:
IntPtr.Zero is more explicit instead of `new IntPtr()` because IntPtr is a value type

Reply:
done